Makefile

```diff
@@ -31,6 +31,14 @@ mexport.so: main.go go.mod
 	$(GOCC) build $(GOFLAGS) -buildmode=plugin -o "$@" "$<"
 	chmod +x "$@"
 
+gen-out:
+	mkdir -p out
+	docker create --name extract $(docker build .)
+	docker cp extract:/usr/local/bin/ipfs ./out/
+	docker rm extract
+	mv out/ipfs/* out
```

Comment:
The script should have changed: `/usr/local/bin/ipfs` in the container is now a single binary, where it used to be a directory containing the kubo binary and the plugin.

Reply:
Are you sure, it still gives me
```
$ ls out
ipfs-v0.17.0-docker  mexport-v0.17.0-docker.so
```
which I guess are IPFS + the plugin?